src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -167,6 +167,7 @@ set_defaults()
   this->only_2D = 0;
   this->kernelised_output_filename_prefix="";
   this->hybrid=0;
+  this->stop_iterative_kernel_at_iter_num=0;
```

Comment:
I think it might be better to set the default to -1 and change the conditions to `<0` instead of `<=0`? I guess a user would never want to fix the kernel at iteration 0, but its technically valid, right? Plus, its more conventional to use a negative "special value" (by special I mean this is either the iteration number or -1 is a special case meaning never).

Reply:
Yes you right. I was unsure about this but I agree. I think the iteration number starts from 1 though so we probably need to trhough an error if the user set it to 0